### PR TITLE
add bias and using standards random initial metric in hessian computation

### PR DESCRIPTION
**Pull request by BiaoFangAIA**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: neural_compressor/adaptor/torch_utils/hawq_metric.py [hunk 1]
```diff
@@ -21,13 +21,17 @@
 
 import copy
 import numpy as np
+from collections import OrderedDict
 import torch.nn
 from torch.quantization.quantize_fx import fuse_fx
+import torch.nn.intrinsic.quantized as nniq
+from torch.fx import symbolic_trace, graph_module
```
Comment (wenhuach21) [inline neural_compressor/adaptor/torch_utils/hawq_metric.py:28]: please remove the import not be used

**Review by wenhuach21 — APPROVED** [review]

File: neural_compressor/adaptor/torch_utils/hawq_metric.py (hunk 1 above)
Comment (BiaoFangAIA) [reply neural_compressor/adaptor/torch_utils/hawq_metric.py]: OK, i removed them and pls review it again
